ui/app/templates/components/job-page/service.hbs

```diff
@@ -13,9 +13,11 @@
     </div>
   </div>
 
-  {{#each this.job.recommendationSummaries as |summary|}}
-    <Das::RecommendationAccordion @summary={{summary}} />
-  {{/each}}
+  {{#if (can "read agent")}}
+    {{#each this.job.recommendationSummaries as |summary|}}
+      <Das::RecommendationAccordion @summary={{summary}} />
+    {{/each}}
+  {{/if}}
```

Comment:
`read agent` is required here? That's surprising to me. Why is that?

Reply:
ughhhhhhhhh just wrong, fixed in ee0e00d 😞